src/common/transformations/tests/symbolic_transformations/nop_broadcast.cpp

```diff
@@ -0,0 +1,111 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/symbolic_transformations/nop_broadcast.hpp"
+
+#include <gtest/gtest.h>
+
+#include <openvino/core/model.hpp>
+#include <openvino/op/broadcast.hpp>
+#include <openvino/op/maximum.hpp>
+#include <openvino/op/parameter.hpp>
+#include <openvino/op/relu.hpp>
+#include <openvino/op/shape_of.hpp>
+
+#include "common_test_utils/ov_test_utils.hpp"
+#include "openvino/core/dimension_tracker.hpp"
+
+using namespace ov;
+using namespace ov::op;
+using namespace std;
+
+namespace {
+void label_shape(ov::PartialShape& shape, size_t start_label = 42) {
+    auto table = std::make_shared<ov::TableOfEquivalence>(42);
+    auto tracker = ov::DimensionTracker(table);
+    tracker.set_up_for_tracking(shape);
+}
+}  // namespace
+
+TEST_F(TransformationTestsF, NopBroadcast1) {
```

Comment:
Minor: I have a small wish regarding test naming. At first It wasn't clear for me what `NopBroadcast1/3` means: is it just test case number or something more. Can we rename it to smth like `NopBroadcastV1/3` or `NopBroadcastOpset1/3`?